index.html

```diff
@@ -1,140 +1,170 @@
-							<a class="button" href="https://github.com/git-for-windows/git/issues" target="_blank">Issue Tracker</a>
-							<h3><span class="gittext">Git for Windows SDK</span> : Contributing Code</h3>
-							<p>The <i>Git for Windows SDK</i> is a build environment that includes all the tools necessary for developers who want to contribute by writing code for Git for Windows.</p>
-							<p>Please look at the <a href="https://github.com/git-for-windows/git/wiki/Technical-overview">technical overview</a> of the Git for Windows packaging and how to include your changes in your own custom installer.</p>
-							<a name="download-sdk" /><a class="button" href="https://github.com/git-for-windows/build-extra/releases/latest" target="_blank">Download <span class="gittext">Git for Windows SDK</span></a>
-							<p>To contribute/update this web page, see its <a href=" https://github.com/git-for-windows/git-for-windows.github.io">Repository</a>.</p>
+					<!-- End Col -->
+					<div class="col-md-6">
+						<div class="footer-menu">
+							<ul>
+								<li>
+									<a href="https://github.com/git-for-windows/git/releases/">All Releases</a>
+								</li>
+								<li>
+									<a href="https://wingit.blob.core.windows.net/files/index.html">Snapshots</a>
+								</li>
+								<li>
+									<a
+										href="https://github.com/git-for-windows/build-extra/blob/master/ReleaseNotes.md#licenses">Licenses</a>
+								</li>
+							</ul>
 						</div>
 					</div>
 				</div>
-				<div class="stud">
-<a href="https://twitter.com/GitForWindows" rel="publisher">Twitter</a>
-				</div>
-			</article>
-		</section>
-		<footer>
-			<div class="content">
-				<ul class="list-unstyled">
-					<li><a style="font-weight:bold" href="https://github.com/git-for-windows/git/wiki/FAQ" target="_blank">FAQ</a></li>
-					<li><a href="https://github.com/git-for-windows/git" target="_blank">Repository</a></li>
-					<li><a href="http://groups.google.com/group/git-for-windows" target="_blank">Mailing List</a></li>
-					<li><a href="https://github.com/git-for-windows/git/releases/">All Releases</a></li>
-					<li><a href="https://wingit.blob.core.windows.net/files/index.html">Snapshots</a></li>
-					<li><a href="https://github.com/git-for-windows/build-extra/blob/master/ReleaseNotes.md#licenses">Licenses</a></li>
-				</ul>
-				<a href="index.html"><h1 class="gittext lowercase">Git for Windows</h1></a>
-				<div class="stud"></div>
 			</div>
-		</footer>
-		<script src="js/pack.js"></script>
-		<script>
-			$(document).ready(function() {
-				$(".fancybox-thumb").fancybox({
-					padding: 0
-				});
+		</div>
+	</footer>
+	<!-- JavaScript files-->
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.0/jquery.slim.min.js"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.3/js/bootstrap.bundle.min.js"></script>
+	<script src="https://cdn.linearicons.com/free/1.0.0/svgembedder.min.js"></script>
```

Comment:
Oops, I mistakenly put this line but, now it will have a purpose in the next iterations.

Reply:
I don't think you need a special JS library to embed SVGs ([source](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Adding_vector_graphics_to_the_Web#the_quick_way_img_element)), so this can still probably be removed.

Reply:
Sure, will see if it works that way.